pkg/kubelet/server/metrics/metrics.go

```diff
@@ -64,6 +65,23 @@ var (
 		},
 		[]string{"method", "path", "server_type", "long_running"},
 	)
+	// HTTPResponse track the response of the http requests.
+	HTTPResponse = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem: kubeletSubsystem,
+			Name:      "http_response_status",
```

Comment:
I don't think we want a new metric for this; rather, we would like to add the response code as a label on existing kubelet metrics. This will allow end users to aggregate metrics by response code on the kubelet.

See e.g. (and note that "code" is one of the included labels)

https://github.com/kubernetes/kubernetes/blob/ee297b6f4d78daaf2f71c78a05203698581ca40a/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L76-L86

Reply:
So if we add the label "code" to the `HTTPRequests`

https://github.com/kubernetes/kubernetes/blob/ee297b6f4d78daaf2f71c78a05203698581ca40a/pkg/kubelet/server/metrics/metrics.go#L33

Will the user call the data of this metrics before receive the change and do some change?  on the other hand, if we add a new label, should they adapt it ?